en/peripherals/camera.md

```diff
@@ -11,25 +11,25 @@ This timestamp can be used for several applications, including: timestamping pho
 
 ## Trigger Configuration
 
-Camera triggering is usually configured from the *QGroundControl* [Vehicle Setup > Camera](https://docs.qgroundcontrol.com/master/en/SetupView/Camera.html#px4-camera-setup) section. 
+Camera triggering is usually configured from the *QGroundControl* [Vehicle Setup > Camera](https://docs.qgroundcontrol.com/master/en/SetupView/Camera.html#px4-camera-setup) section.
 
-![Trigger pins](../../assets/camera/trigger_pins.png) 
+![Trigger pins](../../assets/camera/trigger_pins.png)
```

Comment:
This image shows the AUX pins being set in the QGC interface. Did that UI get fixed now that the config can only be done through Actuators?

Reply:
No, but thanks for the reminder. I'll look into that.

Reply:
I checked and the `Hardware Settings` are only shown if the param exists. So it should be fine already.

Reply:
Excellent. I just cropped the image.